tests/includes/juju.sh

```diff
@@ -327,6 +327,8 @@ post_bootstrap() {
 	controller=${1}
 	model=${2}
 
+	unset BOOTSTRAP_ADDITIONAL_ARGS
```

Comment:
Add a note to say why we want to do this; essentially, noting that bootstrapping twice in one test.